src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+  
+  &__monitor {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  
+  &__description {
+    margin-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main;
+  }
+  
+  &__product-code {
+    color: $color-small;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    font-weight: 400;
+  }
+  
+  &__review-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    
+    align-items: center;
+    margin-bottom: 16px;
+  }
+  
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-small;
+  }
+  
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main;
+  }
+  
+  &__button {
+    width: 166px;
+    height: 40px;
+    left: calc(50% - 166px / 2);
```

Comment:
actually you don't need this `left` property. it's not even active now
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/102411130/215034914-bf0f99dd-3473-46ed-856b-9cbe55731334.png">

Reply:
the same with margin. you have margin-bottom in block above

Reply:
also it's not a link, so it should be a `button` tag